src/odemis/util/transform.py

```diff
@@ -818,3 +818,176 @@ def inverse(self):
         Ainv = numpy.dot(tri_inv(SL), numpy.transpose(self.rotation_matrix))
         tinv = -numpy.dot(Ainv, self.translation)
         return self.__class__(matrix=Ainv, translation=tinv)
+
+
+class AnamorphosisTransform(AffineTransform):
+    """
+    AnamorphosisTransform
+
+    The anamorphosis transform is a polynomial model of the distortions in an
+    electron optical system, and is described in more detail in [1].
```

Comment:
Would it be possible to add a one or two sentences with a bit more explanation? 

If I understand correctly it's like an affine transform and the higher order coefficients allow for inter-coordinate distortion?